### DISCUSSION TITLE
Can oap-server support multipart storage config

### DISCUSSION THREAD

**Discussion by 807801002** [apache/skywalking#7693] [Q&A] [answered]
I want to write metric data to multipart db, just as ES7 for oap-server query for UI, and MySQL for grafana query or bin log subscribe to  synchronize real time to graph db to analyze multipart step data. Can oap server support it?

**Comment by wu-sheng** [comment, marked as answer]
No, this makes no sense. Grafana could read ElasticSearch too. There is no point to duplicate storage load, and provide nothing more, but more risk for inconsistent data.

**Reply by 807801002** [reply]
Grafana query es data is difficult to handle the time bucket field,which is saved as long type and not a timestamp. and es is difficult to handle join query. Can I add insert MySQL at MetricESDAO#prepareBatchInsert and MetricESDAO#prepareBatchUpdate to make it?

**Reply by wu-sheng** [reply]
No, upstream wouldn't accept it. The reason is very clear.
About the timestamp, yes, it hasn't been supported. But that is a simple feature, if someone wants to contribute, a new extra field could be added, like literal names added.
